merge.go

```diff
@@ -61,6 +68,8 @@ type MergeSetConfig struct {
 
 	// new value if its a function, do not include
 	TargetFunction *FunctionConfig
+
+	Type MergeSetConfigType
```

Comment:
Why not directly make the operator configurable?
```suggestion

	// Operator is the operator used in the SET clause. Must be "=" or "+=".
	// If not set, defaults to "=".
	Operator BooleanOperator
```